### DISCUSSION TITLE
posts/hugo-add-component/

### DISCUSSION THREAD

**Discussion by giscus** [leegical/leegical.github.io#4] [Announcements]
# posts/hugo-add-component/

本文主要记录了如何引入 giscus 为博客添加评论功能。用户登陆Github账号后即可评论文章。

https://haoyep.com/posts/hugo-add-component/

**Comment by hzp2333** [comment]
请问博主是怎么实现评论区边框也能切换夜间模式的呢？

**Reply by leegical** [reply]
我并没有为评论区设置任何样式，切换夜间模式可能是由[FixIt主题](https://github.com/hugo-fixit/FixIt)自动实现的

**Comment by Remyy-y** [comment]
支持

**Comment by smallkaka** [comment]
为啥我的博客评论区不显示，但是在github上可以看到我的评论

**Reply by Remyy-y** [reply]
> 为啥我的博客评论区不显示，但是在github上可以看到我的评论

f12，看报错

**Reply by leegical** [reply]
可能是你的hugo主题配置有问题
